media_driver/linux/common/os/libdrm/include/i915_drm_ext.h

```diff
@@ -61,6 +61,41 @@ struct drm_i915_gem_context_param_sseu {
     };
 };
 
+#else
+
+struct drm_i915_gem_create_android {
+    /**
+     * Requested size for the object.
+     *
+     * The (page-aligned) allocated size for the object will be returned.
+     */
+    __u64 size;
+    /**
+     * Returned handle for the object.
+     *
+     * Object handles are nonzero.
+     */
+    __u32 handle;
+    __u32 pad;
+    __u32 flags;
+#define I915_CREATE_PLACEMENT_NORMAL    0 /* standard swappable bo  */
+#define I915_CREATE_PLACEMENT_STOLEN    1 /* Cannot use CPU mmaps */
```

Comment:
I thought you said that STOLEN is not used at the moment? Does that mean we can entirely remove `__flags` right now?

Reply:
So, I suggest to attempt to simply remove __flags and related code. How do you think?